openapi/openapi.json

```diff
@@ -25116,6 +25116,14 @@
         "routingViaHost": {
           "description": "RoutingViaHost allows pod egress traffic to exit via the ovn-k8s-mp0 management port into the host before sending it out. If this is not set, traffic will always egress directly from OVN to outside without touching the host stack. Setting this to true means hardware offload will not be supported. Default is false if GatewayConfig is specified.",
           "type": "boolean"
+        },
+        "v4InternalMasqueradeSubnet": {
+          "description": "V4InternalMasqueradeSubnet contains the v4 masquerade addresses used internally by ovn-kubernetes to enable host to service traffic. The host is configured with these addresses, as well as the shared gateway bridge interface.The values can be changed after installation. The default subnet is 169.254.169.0/29",
```

Comment:
NIT: add a space after period "... gateway bridge interface. The values can be changed after ..."